app/src/main/java/co/nimblehq/compose/ecommerce/ui/screen/home/Home.kt

```diff
@@ -1,36 +1,189 @@
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.BadgeBox
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Surface
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.unit.sp
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import co.nimblehq.compose.ecommerce.R
+import co.nimblehq.compose.ecommerce.ui.theme.AppTextStyle
+import co.nimblehq.compose.ecommerce.ui.theme.Blue
 import co.nimblehq.compose.ecommerce.ui.theme.Purple600
+import co.nimblehq.compose.ecommerce.ui.theme.Red
 
+@ExperimentalMaterialApi
+@OptIn(ExperimentalComposeUiApi::class)
 @Composable
 fun HomeScreen() {
+    val shape = RoundedCornerShape(12.dp)
+
     Column(
         modifier = Modifier
             .fillMaxSize()
+            .padding(16.dp)
+            .background(color = Color.White)
+            .wrapContentSize(Alignment.TopCenter)
             .wrapContentSize(Alignment.Center)
     ) {
-        Text(
-            text = "Home View",
-            fontWeight = FontWeight.Bold,
-            color = Purple600,
-            modifier = Modifier.align(Alignment.CenterHorizontally),
-            textAlign = TextAlign.Center,
-            fontSize = 25.sp
-        )
+        ConstraintLayout(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 50.dp, bottom = 21.dp)
+        ) {
+            val (tvDiscover, bbProfile) = createRefs()
+
+            Text(
+                modifier = Modifier.constrainAs(tvDiscover) {
+                    start.linkTo(parent.start)
+                    top.linkTo(parent.top)
+                },
+                text = stringResource(R.string.home_discover),
+                style = AppTextStyle.title
+            )
+
+            BadgeBox(
+                modifier = Modifier.constrainAs(bbProfile) {
+                    end.linkTo(parent.end)
+                    bottom.linkTo(parent.bottom)
+                },
+                badgeContent = {
+                    Text(
+                        "3",
+                        style = AppTextStyle.badge
+                    )
+                },
+                backgroundColor = Red
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_profile),
```

Comment:
@hoangnguyen92dn Do we have an image for the `SVG` file? If yes, Would it be better if we use it instead of `png`

Reply:
@hoangnguyen92dn But I can download `The SVG` file. Could you please try again?

![Screen Shot 2565-04-26 at 09 25 43](https://user-images.githubusercontent.com/28002315/165207226-f50ea065-7257-4843-9479-b7613fbb0a16.png)

Reply:
@Wadeewee You can export like that's way by selecting `SVG` but when importing to the Android Studio, it shows nothing

![image](https://user-images.githubusercontent.com/6950766/165358320-22b11e95-065d-49dc-937f-3b7121afecce.png)